maintainers/maintainer-list.nix

```diff
@@ -6790,6 +6790,12 @@
     githubId = 4061736;
     name = "Severen Redwood";
   };
+  sstef = {
+    email = "stephane@nix.frozenid.net";
+    github = "fkstef";
+    githubId = 8668915;
+    name = "Stephane Schitter";
+  };
```

Comment:
should be a separate commit
```
maintainers: add sstef
```

Reply:
I moved this part to this pull request: https://github.com/NixOS/nixpkgs/pull/80954